packages/ui-components/src/Tooltip/Tooltip.tsx

```diff
@@ -67,36 +70,48 @@ export const Tooltip: FunctionComponent<TooltipProps> = ({
       ) : (
         (child as ReactElement)
       ),
-      {
-        ref: setReferenceElement,
-        onMouseOver: () => {
-          popperElement.setAttribute("data-show", "");
-        },
-        onMouseOut: () => {
-          if (visible) return;
-          popperElement.removeAttribute("data-show");
-        },
-      },
     );
   });
 
+  const timers: any = [];
```

Comment:
We are trying to keep use of `any` out of the code base. You can replace it like this,

```suggestion
  const timers: Array<ReturnType<typeof setTimeout>> = [];
```